src/navi/transform.clj

```diff
@@ -155,7 +159,9 @@
           types (.getTypes schema)]
       (case (count types)
         nil (throw (IllegalArgumentException. (str "Invalid schema: " schema)))
-        0 (transform-composed schema)
+        0 (if (composed-schema-data schema)
+            (transform-composed schema)
+            :any)
```

Comment:
That is, if the schema doesn't have any types and is not a composed-schema, we default to `:any`. Given that swagger-parser doesn't seem to recognize `anyValue`, I don't think we can do much better than this.